homework/index.js

```diff
@@ -30,14 +36,102 @@
     return elem;
   }
 
+  function createOptions(repoNames, select) {
+    createAndAppend('option', select, { text: 'Select module', disabled: 'disabled' });
+    repoNames.sort((a, b) => a.name.localeCompare(b.name));
+    repoNames.forEach((repository, i) => {
+      createAndAppend('option', select, {
+        text: repository.name,
+        value: repository.name,
+      });
+    });
```

Comment:
As hinted in the assignment, use the array index as the value attribute of the `option` element (it will also get rid of an ESLint error about an unused `i` variable: it is now used):

```js
createAndAppend('option', select, {
  text: repository.name,
  value: i,
});
```

This allows you to directly access the relevant repository information in your event listener for the `<select>` element in function `main()` with:

```js
select.addEventListener('change', () => {
  const repository = repositories[select.value];
  renderRepoInformation(repository, ul);
  renderContributorsInformation(repository, contrList);
});
```

No need for an extra fetch!